### PR TITLE
=str #17173 Clarify CompletionHandling scaladoc

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

**Comment by ktoso** [comment]
refs #17173

**Comment by ktoso** [comment]
LGTM

File: akka-stream/src/main/scala/akka/stream/javadsl/FlexiRoute.scala [hunk 1]
```diff
@@ -92,10 +92,12 @@ object FlexiRoute {
    * handle cancel from downstream output.
    *
    * The `onUpstreamFinish` method is called the upstream input was completed successfully.
```
Comment (drewhk) [inline akka-stream/src/main/scala/akka/stream/javadsl/FlexiRoute.scala]: called when

**Comment by drewhk** [comment]
LGTM

File: akka-stream/src/main/scala/akka/stream/javadsl/FlexiRoute.scala (hunk 1 above)
Comment (patriknw) [inline akka-stream/src/main/scala/akka/stream/javadsl/FlexiRoute.scala]: thx, fixed

**Pull request merged**